x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/ConnectorConstants.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.application.connector;
```

Comment:
should we use the `application` namespace here? I think it's the right decision to use the `ent-search` module, but I am not sure whether we should use `application` here.
`application` is used for search applications and behavioural analytics, I don't think we talked about connectors being in the same category.

Reply:
++

Reply:
That makes sense to me, what about  `ingest` as the namespace?

Reply:
Sounds good to me 👍

Reply:
Okay so as far as I can tell, this means I need to:
- move the `EnterpriseSearch.java` stuff up a level, probably into a new directory called something like `x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/ent-search`
- move `connector` to `ent-search/src/main/java/org/elasticsearch/xpack/ent-search/ingest/connector`
- move `application` to `ent-search/src/main/java/org/elasticsearch/xpack/ent-search/application`

Does that sounds about right?

Reply:
I'm happy to do so, but I imagine that reshuffling may warrant some wider discussion :)

Reply:
Discussed and concluded that we'll revisit this in a later PR.